### PR TITLE
TODO - Add week component

### PR DESCRIPTION
**Pull request by jrojas28**
The following overall features were added:
- Added Week Component
- Added Tests for Week Component
- Added CSS for Week Component
- Integrated Week Component with Month Component
- Small UI Fixes (Font size changes) to Timeslot CSS.
- Other small fixes.

### CODE REVIEW

**Comment by jrojas28** [comment]
[Add Week Component](https://trello.com/c/oHTgfYxf/16-add-week-component)

File: src/js/components/month.jsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import moment from 'moment';
+// import classnames from 'classnames';
```
Comment (lrojas94) [inline src/js/components/month.jsx]: If they are not being used, just remove them

File: src/js/components/month.jsx [hunk 2]
```diff
@@ -35,10 +36,22 @@ export default class Month extends React.Component {
...
+
+    const {
+      weeks,
+    } = this.props;
+
     return (
       <div className = "tsc-month">
         { this._renderTitle() }
         { this._renderActions() }
+        <Week
```
Comment (lrojas94) [inline src/js/components/month.jsx]: Please put this inside a function `_renderWeek()`

File: src/js/components/month.jsx [hunk 3]
```diff
@@ -67,7 +80,7 @@ export default class Month extends React.Component {
     const currentWeek = weeks[currentWeekIndex];
     const startDate = helpers.getMomentFromCalendarJSDateElement(currentWeek[0]);
     const endDate = helpers.getMomentFromCalendarJSDateElement(currentWeek[currentWeek.length - 1]);
-    const actionTitle = `${startDate.format('Do')} - ${endDate.format('Do')}`;
+    const actionTitle = `${startDate.format('MMM Do')} - ${endDate.format('MMM Do')}`;
```
Comment (lrojas94) [inline src/js/components/month.jsx]: Great o/

File: src/js/components/week.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <Day
+          key = { index }
+          onTimeslotClick = { this._onTimeslotClick.bind(this, index) }
+          momentTime = { formattedDate }
+          />
+      );
+    });
+  }
+
+  _onTimeslotClick(index) {
```
Comment (lrojas94) [inline src/js/components/week.jsx]: This doesn't make any sense, why would I want the index to go up? 

From `Timeslot`:
 -> onTimeslotClick is called with `index`

From `Day`: 
-> takes the index sent by `Timeslot` and looks into it's array. Generates a DateTime string which is passes to `Week`. e.g: `Jun 14th 2015 - 2PM`

From Week:
-> Pass that info to month - doesn't need to touch it

From Month:
-> Pass that info to Calendar - Which adds it to the state.

**Pull request merged**